### DISCUSSION TITLE
Does the client support MAC for the m1 chip?

### DISCUSSION THREAD

**Discussion by yichangqiao** [SoftEtherVPN/SoftEtherVPN#2010] [Q&A]
I want to use the m1 chip ARM architecture on a computer and install the software client

**Comment by chipitsine** [comment]
I would suggest MacPort: https://ports.macports.org/port/softether5/  (or building from source).
unfortunately there's no many users on Mac, I guess there might be issues


we would appreciate if you could try installing on Mac m1 and give some feedback

**Reply by yichangqiao** [reply]
Is this applicable for client certificate authentication?
